libmodernize/main.py

```diff
@@ -17,6 +18,52 @@
 from libmodernize import __version__
 from libmodernize.fixes import lib2to3_fix_names, six_fix_names, opt_in_fix_names
 
+
+PY3K = sys.version_info >= (3, 0)
+if not PY3K:
+    LF = '\n'
+    CRLF = '\r\n'
+    CR = '\r'
+else:
+    LF = bytes('\n', encoding='ascii')
```

Comment:
Just use the b prefix, like `b'\n'`. It's valid syntax on Python 2.6 and above, which is what we support. And then you don't need an if/else, because on Python 2, `b'\n' == '\n'`.